scripts/provisioning/cleanupcmd.py

```diff
@@ -115,31 +112,32 @@ def delete_ldap_account(self, ldap_acc: str):
       s3cipher_obj = CortxS3Cipher(None, True, 22, acc_attributes_dict["cipherConstKey"])
```

Comment:
Should not need to use cipher.
LDAPsearch should retrive the same - investigate